README.md

```diff
@@ -53,18 +53,20 @@ pseudo_pop <- generate_pseudo_pop(Y,
                                   gps_model = "parametric",
                                   use_cov_transform = TRUE,
                                   transformers = list("pow2", "pow3"),
-                                  sl_lib = c("m_xgboost","SL.earth","SL.gam",
-                                             "SL.ranger"),
-                                  params = list(xgb_nrounds=c(10,20,30),
-                                                xgb_eta=c(0.1,0.2,0.3)),
+                                  sl_lib = c("m_xgboost"),
+                                  params = list("xgb_nrounds" = 50,
```

Comment:
Please drop "" in the list. For example, "xgb_nrounds" = 50. --> xgb_nrounds = 50.